### DISCUSSION TITLE
Using multiple modifiers

### DISCUSSION THREAD

**Discussion by ColonelBlimp** [tailwindlabs/tailwindcss#6834] [Help] [answered]
Hi,
I want to use a media query modifier (`sm:`) along with the hover modifier (`hover:`) - something like `sm:hover:border-b-2`

Any ideas about how to do this?

Many thanks.

**Comment by alexanderbluhm** [comment, marked as answer]
It is possible like you already assumed, so `sm:hover:border-b-2` works!
You can check this [link](https://play.tailwindcss.com/DAKlJ56oex) for a demo.
Which Tailwind version do you use?

**Reply by ColonelBlimp** [reply]
Version 3.0.8.  I tried this but it didn't work, but then I am also using the latest version of Hugo, and tailwindcss JIT and Hugo are not playing nicely together.  I am ironing out the niggles, but...

Thank you for the help.  Appreciated!
